_harp/apply/index.ejs

```diff
@@ -2,4 +2,8 @@
 	<div class="container apply-page">
 		<%- partial("_apply.md") %>
 	</div>
+	<div align="center" class="programmes-info">
+		<a href="/apply/london"><button class="button-three">LONDON</button></a>
+		<a href="/apply/nazareth"><button class="button-three">NAZARETH</button></a>
```

Comment:
You to scroll to find these buttons but people might skip the pre-reqs if we move them to the top... 
@jsms90 , @emilyb7 , thoughts?